quantized_mesh_encoder/ecef.py

```diff
@@ -1,9 +1,9 @@
 import numpy as np
 
-from .constants import WGS84_A, WGS84_E2
+from .constants import WGS84
```

Comment:
I think it'll be nice to use a dataclass for this. E.g.

```py
@dataclass
class Ellipsoid:
	a: float
	b: float

WGS84 = Ellipsoid(a=6378137.0, b=6356752.3142451793)
```

and then use `ellipsoid.a` instead of `ellipsoid['a']`, and you can also `assert isinstance(ellipsoid, Ellipsoid), 'ellipsoid must be an instance of the Ellipsoid class'`